modules/environment/README.md

```diff
@@ -15,6 +15,7 @@ ZSH Options
 | AUTO_RESUME | Resume existing jobs before creating a new job |
 | LONG_LIST_JOBS | Use the verbose list format by default |
 | NOTIFY | Report job status immediately instead of waiting for new prompt |
+| INTERACTIVECOMMENTS | Recogniz `#` as comments
```

Comment:
@Eriner, I think you can make a better suggestion for the description here...  :- )